docs/source/upgrade-guide/index.md

```diff
@@ -48,7 +48,64 @@ You should take the following actions for your Volto 19 projects.
 
 If you can't upgrade immediately, you may continue to run Volto 19 on Node.js 20 at your own risk, but be aware that issues specific to Node.js 20 will not be fixed in the Volto core CI or releases.
 
-(19-removed-support-for-loading-configuration-from-project-label)=
+### Changes to the project frontend folder boilerplate
```

Comment:
We already had this: https://github.com/plone/volto/blob/main/docs/source/upgrade-guide/index.md#pnpm-10-no-longer-runs-lifecycle-scripts

Perhaps this could be removed?

Reply:
Or merged with it? It does appear to be somewhat redundant.

Reply:
@wesleybl I think that is better to merge, and explain well what is this feature doing.

Reply:
I just merged them.